docs/Getting Started Tutorial.md

```diff
@@ -0,0 +1,267 @@
+    #     ___  ____     _    ____ _     _____
+    #    / _ \|  _ \   / \  / ___| |   | ____|
+    #   | | | | |_) | / _ \| |   | |   |  _|
+    #   | |_| |  _ < / ___ | |___| |___| |___
+    #    \___/|_| \_/_/   \_\____|_____|_____|
+***
+# The getting started tutorial
+
+## Goal
+
+Using the Oracle Cloud Infrastructure (OCI) Terraform provider, we would like to set up a basic infrastructure, but with all the typical components.
+
+We would build a network from scratch, spin up compute instances, set up some storage etc.
+
+## Intended audience
+
+We hope this tutorial can be used by people who are not familiar with Terraform or the Oracle Cloud Infrastructure (OCI).
+
+This tutorial would provide some hands-on experience in setting up your infrastructure on the OCI platform.
+
+This tutorial may also be used as sample code to start out with when building your own infrastructure on OCI.
+
+## Install Terraform
+
+In order to install Terraform on your machine, follow the straightforward steps as described [here](https://www.terraform.io/intro/getting-started/install.html).
+
+Verify that your installation works as suggested in the linked page above.
+
+Run the following command -
+
+```bash
+terraform version
+```
+
+Ensure that the Terraform version is `v0.10.x` or higher.
```

Comment:
Is this just so that we don't have to include the extra instructions for earlier versions?